Azure-Samples/azure-intelligent-edge-patterns
# updating readme for kubeflow on azure stack tutorial

## Purpose
* Addressing the feedback from people following the instructions, adding a few helpful details.

## Does this introduce a breaking change?
```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
```
Follow the instructions in the documentation, and you should have a Kubeflow on Azure Stack at the end.
```

## What to Check
Verify that the following are valid
* see above.

## Other Information
n/a
